src/python/review/common/file_system.py

```diff
@@ -66,8 +67,9 @@ def create_file(file_path: Union[str, Path], content: str):
     file_path = Path(file_path)
 
     create_directory(os.path.dirname(file_path))
-    with open(file_path, 'w') as f:
-        f.write(content)
+    with open(file_path, 'w+') as f:
+        f.writelines(content)
+        yield Path(file_path)
 
 
 def create_directory(directory: str) -> None:
```

Comment:
Why do you need this alias function? It's simpler and better to just use `os.makedirs` when needed.

Reply:
I think we should keep this function. We have a function that creates a file, so why can't there be a function that creates a directory?